fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -10,8 +10,23 @@ def self.run(params)
 
         skip_git_hooks = params[:skip_git_hooks] ? '--no-verify' : ''
 
-        result = Actions.sh("git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip)
-        UI.success("Successfully committed \"#{params[:path]}\" 💾.")
+        command = "git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip
+        nothing_to_commit = false
+
+        if params[:allow_nothing_to_commit]
+          result = Actions.sh(
+            command,
+            log: FastlaneCore::Globals.verbose?,
+            error_callback: lambda { |_|
```

Comment:
This catching of error callback might be a bit to aggressive. It _will_ allow nothing to get committed but it will also eat all/any other errors as well 🤔 Might need to look into another solution 🙃